index.html

```diff
@@ -49,5 +49,191 @@ <h1 id="name">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="work-tools">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+        </div>
+        <div class="portfolio">
+            <div class="project">
+                <div class="work-img">
+                    <img src="/assets/images/projects/tonic.png" alt="tonic image">
+                </div>
+                <div class="work-description">
+                    <h2 class="work-title">tonic</h2>
+                    <ul class="canopy">
+                        <li>canopy</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="work-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="work-tools">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+        </div>
+        <div class="portfolio">
+            <div class="project">
+                <div class="work-img">
+                    <img src="/assets/images/projects/anoter-one.png" alt="tonic image">
+                </div>
+                <div class="work-description">
+                    <h2 class="work-title">Multi-Post Stories</h2>
+                    <ul class="canopy">
+                        <li>canopy</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="work-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="work-tools">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section id="about">
+        <div class="about-container">
+            <h1 class="name">About Myself</h1>
+            <p class="details">
+                Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <a href="#" class="connect">let's connect</a>
```

Comment:
- [ ] Consider using an `h` tag for this to follow best practices. we use `a` tag only when there is a link or button.